pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -352,6 +352,7 @@ def patch(pkg_root: Path, srcpath: Path, src_metadata: dict[str, Any]):
             result = subprocess.run(
                 ["patch", "-p1", "--binary", "--verbose", "-i", pkg_root / patch],
                 check=False,
+                encoding="utf-8",
```

Comment:
These three paths were actually buggy, the error printer assumed strings and not bytes.